djangocms_moderation/admin_actions.py

```diff
@@ -10,9 +10,9 @@
 
 from cms.utils.urlutils import add_url_parameters
 
-from django_fsm import TransitionNotAllowed
 from djangocms_versioning.models import Version
 
+from django_fsm import TransitionNotAllowed
```

Comment:
This import seems to be out of order now. It should be above versioning.

Reply:
Same here. Isort complains if I move it.